provision/acc_provision/templates/nettools-manifests.yaml

```diff
@@ -0,0 +1,319 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: connectivitycheckers.nettools.debug
+spec:
+  group: nettools.debug
+  names:
+    kind: ConnectivityChecker
+    listKind: ConnectivityCheckerList
+    plural: connectivitycheckers
+    singular: connectivitychecker
+    shortNames:
+    - conncheck
+    - reachcheck
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    served: true
+    storage: true
+    schema:
+      openAPIV3Schema:
+        description: ConnectivityChecker defines the reachability test results of all pods.
+        properties:
+          apiVersion:
+            type: string
+          kind:
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: ConnectivityCheckerSpec defines the results of various reachability tests.
+            properties:
+              reachability_test_enable:
+                type: object
+                properties:
+                  periodic_sync:
+                    type: object
+                    properties:
+                      enable_updates:
+                        type: boolean
+                      interval:
+                        type: integer
+                        default: 120
+                  external_url:
+                    type: object
+                    properties:
+                      url:
+                        type: string
+                        default: 'google.com'

```

Comment:
Can we check which URL is used by the upstream kubernetes e2e conformance tests?

Reply:
kubernetes e2e conformance tests also use same URL google.com https://github.com/kubernetes/kubernetes/blob/master/test/e2e/network/networking.go#L102